index.html

```diff
@@ -0,0 +1,215 @@
+          </li>
+          <li>
+            <a href="./about.html">about</a>
+          </li>
+          <li>
+            <a href="">program</a>
+          </li>
+          <li>
+            <a href="">join</a>
+          </li>
+          <li>
+            <a href="">sponsor</a>
+          </li>
+          <li id="larger">
+            <a href="">NTS campaign</a>
+          </li>
+
+          <li class="icon">
+            <i class="fa-sharp fa-solid fa-xmark"></i>
+          </li>
+
+
+        </ul>
+
+      </nav>
+
+    </main>
+            
+    <main class="intro">
+      <h3>"Hello! you're welcome"</h3>
+      <h1> To annual national<br>tech summit<br> 2022</h1>
+      <p class="bg"> celebrating the openess, creativity in the tech community and sharing the different ethnicity of people from different part of the country
+        under one space </p>
+      <h2>2022.11.8(tue) ~ 9(wed)</h2>
+      <p>at 294 herbert macaulay way, yaba 101212, lagos</p>
+    </main>
+  
+  </header>
+
+  <section class="program">
+
+    <h2>main program</h2>
+
+    <div class="container">
+
+      <main class="pros">
+
+        <div class="img">
+          <i class="fa-solid fa-chalkboard"></i>
+          <h4>lecture</h4>
+        </div>
+
+        <p>
+          Listen to different speakers from across the country and outside the country.
+        </p>
+      </main>
+
+      <main class="pros">
+        <div class="img">
+          <i class="fa-solid fa-people-arrows"></i>
+            <h4>exhibition</h4>
+        </div>
+
+        <p>
+          Apperciate different creativity and craft from graphic designers and nft makers
+        </p>
+      </main>
+
+      <main class="pros">
+        <div class="img">
+          <i class="fa-solid fa-comments"></i>
+            <h4>forum</h4>
+        </div>
+        <p>
+          A space to chat, learn ,share thoughts and opinion with peer from across the country
+        </p>
+      </main>
+
+      <main class="pros">
+        <div class="img">
+          <i class="fa-solid fa-toolbox"></i>
+          <h4>workshop</h4>
+        </div>
+        <p>
+          Try creating your own work making use of open-source license rather than watching it.
+        </p>
+      </main>
+
+      <main class="pros">
+        <div class="img">
+          <i class="fa-solid fa-network-wired"></i>
+          <h4>NTS ignite</h4>
+        </div>
+        <p>
+          Get opportunities to network with NTS affliate around the the country and world even after the summit 
+        </p>
+      </main>
+
+    </div>
+
+    <div class="btn">
+      <button> join NTS 2022</button>
+      <a href="">see more programs</a>
+    </div>
+        
+  </section>
+  
+  <section class="speaker">
+    <div class="contain">
+      <h2>featured speakers</h2>
+      <div class="wrapper">
+      </div>
+    </div>
+    <div class="more">
+      <button>
+        <span>more</span> 
+         <i class="fa-solid fa-angle-down"></i>
```

Comment:
- [ OPTIONAL ] Please make sure you add some Javascript to display a longer list when the user clicks on the more button :recycle: 
- [OPTIONAL] Kindly make sure the small red background at the top left of the speakers' image is a grey-white chess image. :recycle: 
__Please note that these requirements are optional only because the previous reviewer forgot to mention them, but they are crucial for this project to be complete__

- Your design:
![More](https://user-images.githubusercontent.com/97834160/210820068-95f35a0d-3bda-4043-a11f-127d84026612.png)

- Behance design:
![moreB](https://user-images.githubusercontent.com/97834160/210820280-f6665ecd-a84c-4b04-a864-46d334079b82.png)